app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -61,59 +56,65 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
-        View currView = inflater.inflate(R.layout.fragment_feed, container, false);
+        // prevent reloading if not necessary
+        if(currView == null) {
+            // Inflate the layout for this fragment
+            currView = inflater.inflate(R.layout.fragment_feed, container, false);
 
-        // find Recycler View
-        rvFeed = currView.findViewById(R.id.rvFeed);
+            // find Recycler View
+            rvFeed = currView.findViewById(R.id.rvFeed);
 
-        // initialize the array that will hold posts and create a PostsAdapter
-        showsList = new ArrayList<>();
-        adapter = new ShowAdapter(getActivity(), showsList);
+            // initialize the array that will hold posts and create a ShowAdapter
+            showsList = new ArrayList<>();
+            adapter = new ShowAdapter(getActivity(), showsList, false);
```

Comment:
Yeah, boolean flags, not great especially with Java which is a language without named parameters -- what does this false mean?  We really don't know.  That's why boolean flags aren't great